### PR TITLE
Adding vuid 01409

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: tests/vklayertests_pipeline_shader.cpp [hunk 1]
```diff
@@ -11659,3 +11659,27 @@ TEST_F(VkLayerTest, ShaderAtomicInt64) {
         *this, set_info, kErrorBit,
         std::vector<string>{"VUID-VkShaderModuleCreateInfo-pCode-01091", "UNASSIGNED-CoreValidation-Shader-AtomicFeature"});
 }
+
+TEST_F(VkLayerTest, PipelineInvalidAdvancedBlend) {
+    TEST_DESCRIPTION("Create a graphics pipeline with advanced blend when its disabled");
+
+    ASSERT_NO_FATAL_FAILURE(Init());
+    ASSERT_NO_FATAL_FAILURE(InitRenderTarget());
+
```
Comment (ncesario-lunarg) [inline tests/vklayertests_pipeline_shader.cpp:11668]: Do you need to make sure `advancedBlendAllOperations == VK_FALSE` for this device?

**Review by ncesario-lunarg — COMMENTED** [review]
One question in the test, but otherwise LGTM!

File: tests/vklayertests_pipeline_shader.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply tests/vklayertests_pipeline_shader.cpp:11668]: I assumed if I don't query it, it will be disabled, but I have updated the test now.

Comment (ncesario-lunarg) [reply tests/vklayertests_pipeline_shader.cpp:11668]: I think that's true for _features_, but I'm not sure if that is also true for _properties_ (which cannot be "enabled"/"disabled").

**Review by ncesario-lunarg — APPROVED** [review]

**Pull request merged**